cmd/atc/resources.go

```diff
@@ -174,6 +174,37 @@ func ApplyResources(ctx context.Context, client *k8s.Client, cfg *Config) (err e
 		return fmt.Errorf("failed to apply airway crd: %w", err)
 	}
 
+	// Get timeout value for airway validation webhook, defaulting to 10 seconds if not configured
+	airwayTimeoutSeconds := func() *int32 {
```

Comment:
perhaps wrap this logic of checking an `int32` and returning a default if <=0 into a small utility function?
This is taking up a lot of vertical space for something that could probably be read inline in a declarative way:

Example:
```go
Timeout: withDefault(cfg.AirwayValidationWebhookTimeout, 10)
```